src/main/resources/application.yaml

```diff
@@ -21,6 +21,17 @@ management:
       enabled: true
     prometheus:
       enabled: true
+rhsm-subscriptions:
+  datasource:
+    # This can be overridden elsewhere in configuration, but it's important to provide a sensible default.
+    # Without this setting, postgres' JDBC driver defaults to using $HOME/.postgresql/root.crt
+    root-ca-location: classpath:/rhsm-subscriptions-ca.crt
```

Comment:
Can we put this in the class instead, as a default for `PostgresTlsDataSourceProperties#rootCaLocation`? That way we wouldn't have to override for every new datasource...

Reply:
Sure.  I don't have a problem with that.